Backend/dev/migrations/dynamic-offer-driver-app/0344-added-govtData-rc-table.sql

```diff
@@ -0,0 +1,29 @@
+ALTER TABLE atlas_driver_offer_bpp.merchant_service_usage_config ADD COLUMN verification_providers_priority_list text[] Not NULL DEFAULT '{"Idfy"}';
+
+UPDATE atlas_driver_offer_bpp.merchant_service_usage_config SET verification_providers_priority_list='{"GovtData", "Idfy"}'; -- Do not run this in prod
+
+CREATE TABLE atlas_driver_offer_bpp.govt_data_r_c  (
+    id VARCHAR(36) PRIMARY KEY,
+    merchant_operating_city_id VARCHAR(36) NOT NULL,
+    owner_serial_number TEXT,
+    registration_number TEXT,
+    manufacturer_model TEXT,
+    permit_validity_from TEXT,
+    permit_validity_upto TEXT,
+    manufacturer TEXT,
+    body_type TEXT,
+    number_of_cylinder INT,
+    fuel_type TEXT,
+    seating_capacity INT,
+    from_date TEXT,
+    to_date TEXT,
+    created_at timestamp with time zone NOT NULL DEFAULT now()
+);
+
+Alter table atlas_driver_offer_bpp.vehicle_registration_certificate alter Column vehicle_capacity set data type integer USING vehicle_capacity::integer ; -- Do not run this in master and prod
```

Comment:
Why not in master and prod?

Reply:
because it's already there in master and prod, someone forgot to add this in local.